docs/data-management/manual-compaction.md

```diff
@@ -146,6 +146,8 @@ The compaction task has two kinds of `inputSpec`:
 
 ## Compaction transform spec
 
+Only filters are supported for compaction transform specs. If you include something other than a filter, such as an expression, Druid ignores the field.
```

Comment:
The other sentence can be left out as it is not really needed.
It is also slightly incorrect since filters can have expressions too.

```suggestion
Only filters are supported for compaction transform specs.
```

Reply:
I think this is trying to mention that `transforms` cannot be set like they can on other transform specs which have both `transforms` and `filters`, which does seem useful to call out. i agree it is slightly confusing to refer to them as expressions here...  though expressions are the only out of the box type of transforms so maybe it is generally confusing 🙃.

Reply:
Yeah, it is confusing to have `filters` inside the `transformSpec`.

Maybe we should phrase it along the lines of `The compaction transformSpec can be used only to specify filters and not transforms` and link to the transformSpec documentation.